tests/components/insteon/test_config_flow.py

```diff
@@ -271,71 +216,31 @@ async def test_import_plm(hass: HomeAssistantType):
     assert result["type"] == "create_entry"
     assert hass.config_entries.async_entries(DOMAIN)
     for entry in hass.config_entries.async_entries(DOMAIN):
-        assert entry.data == MOCK_USER_INPUT_PLM
-
-
-async def test_import_plm_full(hass: HomeAssistantType):
-    """Test importing a full PLM config from yaml."""
-    await setup.async_setup_component(hass, "persistent_notification", {})
-    result = await _import_config(hass, MOCK_IMPORT_FULL_CONFIG_PLM)
-    assert result["type"] == "create_entry"
-    assert hass.config_entries.async_entries(DOMAIN)
-    for entry in hass.config_entries.async_entries(DOMAIN):
-        assert entry.data == MOCK_USER_INPUT_PLM
-        assert entry.options[CONF_OVERRIDE][0][CONF_ADDRESS] == "1A.2B.3C"
-        assert entry.options[CONF_OVERRIDE][0][CONF_CAT] == MOCK_CAT
-        assert entry.options[CONF_OVERRIDE][0][CONF_SUBCAT] == MOCK_SUBCAT
-        assert entry.options[CONF_X10][0][CONF_HOUSECODE] == MOCK_HOUSECODE
-        assert entry.options[CONF_X10][0][CONF_UNITCODE] == MOCK_UNITCODE
-        assert entry.options[CONF_X10][0][CONF_PLATFORM] == MOCK_X10_PLATFORM
-        assert entry.options[CONF_X10][0][CONF_DIM_STEPS] == MOCK_X10_STEPS
-
-
-async def test_import_full_hub_v1(hass: HomeAssistantType):
-    """Test importing a full Hub v1 config from yaml."""
-    await setup.async_setup_component(hass, "persistent_notification", {})
+        assert entry.data == MOCK_IMPORT_CONFIG_PLM
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)